bigbluebutton-html5/imports/startup/server/redis.js

```diff
@@ -159,6 +159,9 @@ class RedisPubSub {
 
     if (ignoredMessages.includes(channel)
       || ignoredMessages.includes(eventName)) {
+      if (eventName === 'CheckAlivePongSysMsg') {
```

Comment:
Would be better to add it to the ignoredMessages list instead of add another condition.

Reply:
Adding it to the ignoredMessages list still shows `CheckAlivePongSysMsg skipped`. My preference was to not show anything on ping-pong